packages/contracts/src/networks.ts

```diff
@@ -24,6 +111,11 @@ export type SupportedNetwork = {
     provider: RollupProvider
     type: RollupType
   }
+  handleNetworkSpecificMerkleLeafGas?: (
+    foundryGas: string,
+    deployedContractSizes: DeployedContractSize[],
+    access: ParsedAccountAccess
+  ) => string
```

Comment:
There seems to be a mismatch between the type definition for `handleNetworkSpecificMerkleLeafGas` and the parameters that it's actually invoked with. This definition doesn't have the `chainId` as the first parameter, but this function is called with the chain ID as the first parameter throughout the codebase

Reply:
This also seems to apply to the definition for `calculateActionLeafGasForMoonbeam`

Reply:
There were actually two functions with the same name which is obviously not ideal.